frontend/src/pages/LearningCourse/CourseInfo/styled.ts

```diff
@@ -0,0 +1,84 @@
+import styled from 'styled-components';
+
+import theme from 'themeSettings';
+
+interface CourseInfoWrapperProps {
+  isCourseInfoOpen: boolean;
+}
+
+export const CourseInfoWrapper = styled('div')<CourseInfoWrapperProps>(({ isCourseInfoOpen }) => ({
+  display: 'inline-block',
+  float: 'left',
+  width: 'calc(100% - 196px)',
+  maxHeight: '438px',
+  paddingBottom: '20px',
+  [theme.breakpoints.down('xl')]: {
+    width: 'calc(100% - 193px)',
+  },
+  [theme.breakpoints.down('md')]: {
+    display: 'block',
+    float: 'none',
+    width: '100%',
+    marginTop: '23px',
+    maxHeight: '290px',
+    paddingBottom: '0px',
+    ...(!isCourseInfoOpen && {
+      display: 'none',
+    }),
+  },
+}));
+
+export const CourseInfoContent = styled('div')({
+  display: 'block',
+  width: 'calc(100% - 196px)',
+  padding: '31px 238px 31px 54px',
+  textAlign: 'left',
+  color: 'black',
+  border: '1px solid #E0E0E3',
+  borderRadius: '8px',
+  overflowY: 'auto',
+  [theme.breakpoints.down('xl')]: {
+    width: '100%',
+    padding: '24px 44px 31px 24px',
+  },
+  [theme.breakpoints.down('md')]: {
+    display: 'block',
+    width: '100%',
+    padding: '8px 19px 3px 8px',
+  },
+});
+
+export const CourseInfoTitle = styled('span')({
+  fontFamily: '"Ubuntu", sans-serif',
```

Comment:
Do we really need to define fontFamily in all the components styles? It's already declared in theme and can be declared for parent component only to avoid extra code